src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseHandler.cs

```diff
@@ -301,6 +301,16 @@ public override Task<InitializeViewResult> InitializeObjectView(InitializeViewRe
                                     // Files tab is only supported in SQL Server, but files exists for all servers and used in detach database, cannot depend on files property to check the supportability
                                     ((DatabaseInfo)databaseViewInfo.ObjectInfo).IsFilesTabSupported = true;
                                     ((DatabaseInfo)databaseViewInfo.ObjectInfo).Filegroups = GetFileGroups(smoDatabase, databaseViewInfo);
+                                    try
+                                    {
+                                        // ServerFilestreamAccessLevel uses xp_instance_regread so will throw if that isn't available. In that case we just default to Disabled to ensure the UI reflects that
+                                        databaseViewInfo.ServerFilestreamAccessLevel = dataContainer.Server.FilestreamLevel;
+                                    }
+                                    catch (PropertyCannotBeRetrievedException ex) {
+                                        // Setting filestream access level to disabled
+                                        databaseViewInfo.ServerFilestreamAccessLevel = FileStreamEffectiveLevel.Disabled;
+                                        Logger.Error(ex.Message);
```

Comment:
Usually it's nice to leave context about the error - for example "Cannot retrieve Server FilestreamLevel, defaulting to Disabled. Error : {ex.Message}"

Reply:
@ssreerama Sorry - I meant in the actual log message itself. The log contains a LOT of information (especially with sqlclient logs added) - and so adding context is super important there so it isn't just some random error message in the middle of a bunch of other stuff.